lib/tasks/dspace.rake

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
```

Comment:
I apologize for including this if it is extraneous; please let me know if it is preferred that I remove this from the PR.

Reply:
No I think Figgy's rubocop forces it right now.